.github/workflows/pull-request-workflow.yml

```
@@ -62,3 +62,33 @@ jobs:
         run: yarn install --immutable
       - name: Build
         run: yarn build
+
+  tests:
+    needs:
+      - build
+    runs-on: ubuntu-latest
+    container: node:16-alpine3.15
```

Can we use matrix here?

```suggestion
    strategy:
      matrix:
        node-version: [ 16.x ]
```

---

Yes, we can do it directly on job runner machine

---

Because, I think it's a handy way to specify what version of nodejs we need to verify and support comparing with docker container where I guess we need to create different jobs for each nodejs version

---

Because, I think it's a handy way to specify what version of nodejs we need to verify and support comparing with docker container where I guess we need to create different jobs for each nodejs version

---

Ok, no problem.